code/core/src/manager/components/layout/Layout.tsx

```diff
@@ -213,33 +235,34 @@ const LayoutContainer = styled.div<LayoutState & { showPanel: boolean }>(
       colorScheme: 'light dark',
 
       [MEDIA_DESKTOP_BREAKPOINT]: {
-        display: 'grid',
-        gap: 0,
-        gridTemplateColumns: `minmax(0, ${navSize}px) minmax(${MINIMUM_CONTENT_WIDTH_PX}px, 1fr) minmax(0, ${rightPanelWidth}px)`,
-        gridTemplateRows: `1fr minmax(0, ${bottomPanelHeight}px)`,
-        gridTemplateAreas: (() => {
-          if (!showPanel) {
-            // showPanel is false by default when viewMode is not 'story', but can be overridden by the user
-            return `"sidebar content content"
-                  "sidebar content content"`;
-          }
-          if (panelPosition === 'right') {
-            return `"sidebar content panel"
-                  "sidebar content panel"`;
-          }
-          return `"sidebar content content"
-                "sidebar panel   panel"`;
-        })(),
+        flexDirection: 'row',
       },
     };
   }
 );
 
-const SidebarContainer = styled.div(({ theme }) => ({
+const SidebarContainer = styled.div<{ width: number }>(({ theme, width }) => ({
   backgroundColor: theme.appBg,
-  gridArea: 'sidebar',
   position: 'relative',
   borderRight: `1px solid ${theme.appBorderColor}`,
+  width,
+
+  [MEDIA_DESKTOP_BREAKPOINT]: {
+    flexShrink: 0,
+  },
+}));
```

Comment:
This would create a new class for every pixel value of `width`.

So though we CAN do it this way, it's preferred to set these types of runtime styling that change a lot, and are exact as inline-styles on the element.